mne/viz/misc.py

```diff
@@ -450,9 +450,10 @@ def plot_bem(subject=None, subjects_dir=None, orientation='coronal',
     -----
     Images are plotted in MRI voxel coordinates.
 
-    For a given slice index, all source points are shown that are halfway
-    between the previous slice and the given slice, and halfway between the
-    current slice and the next slice. For large slice decimations, this can
+    If ``src`` is not None, for a given slice index, all source points are
+    shown that are halfway between the previous slice and the given slice,
+    and halfway between the current slice and the next slice.
```

Comment:
here you refer to both "given slice" and "current slice".  Should call it the same thing (either seems fine).